src/renderer/components/message/Message.tsx

```diff
@@ -586,6 +589,11 @@ export default function Message(props: {
   const hasText = text !== null && text !== ''
   const fileMime = (!isSetupmessage && message.fileMime) || null
   const isWithoutText = isMediaWithoutText(fileMime, hasText, message.viewType)
+  const showAttachment = (message: T.Message) =>
```

Comment:
`showAttachment` as function sounds like an action, maybe rename it to sth like `shouldShowAttachment` or make it a variable/const instead of a function?